support-frontend/assets/components/checkoutBenefits/benefitsCheckList.tsx

```diff
@@ -80,6 +80,7 @@ export type BenefitsCheckListData = {
 	strong?: boolean;
 	isNew?: boolean;
 	hideBullet?: boolean;
+	label?: string;
```

Comment:
Perhaps `pill` might describe what component will be having its copy  changing? Mind can see `label` is neat.

Reply:
👍 i'll change it to pill here